sdk/attestation/azure-security-attestation/README.md

```diff
@@ -127,15 +127,15 @@ clients to add, remove or enumerate the policy management certificates.
 
 ## Examples
 
-- [Instantiate a synchronous attestation client](#create-a-synchronous-attestation-client)
-- [Retrieve token validation certificates](#retrieve-token-certificates)
+- [Create a synchronous attestation client](#create-a-synchronous-attestation-client)
+- [Retrieve Token Certificates](#retrieve-token-certificates)
 - [Attest an SGX enclave](#attest-an-sgx-enclave)
-- [Instantiate a synchronous administrative client](#create-a-synchronous-administrative-client)
-- [Get attestation policy](#retrieve-current-attestation-policy-for-openenclave)
+- [Create a synchronous administrative client](#create-a-synchronous-administrative-client)
+- [Retrieve current attestation policy for OpenEnclave](#retrieve-current-attestation-policy-for-openenclave)
 - [Set unsigned attestation policy](#set-unsigned-attestation-policy-aad-clients-only)
 - [Set signed attestation policy](#set-signed-attestation-policy)
-- [List policy management certificates](#list-attestation-signing-certificates)
-- [Add policy management certificate](#add-attestation-signing-certificate)
+- [List attestation signing certificates](#list-attestation-signing-certificates)
```

Comment:
These are NOT attestation signing certificates. They aren't used to sign anything, they're used to verify that the client is in possession of a key which authorizes their ability to modify administrative properties on the instance in an Isolated Mode attestation service instance..

The terminology used for this functionality in the Java SDK is "policy management certificate". And this API returns those certificates.